### DISCUSSION TITLE
Any large companies using stirling-pdf?

### DISCUSSION THREAD

**Discussion by dabump** [Stirling-Tools/Stirling-PDF#467] [General]
Stirling-pdf looks great! Was wondering if anybody knows of larger companies backing / using stirling-pdf?

**Comment by Frooodle** [comment]
There are many universities that I know of that use it and various small offices. Cloudron also supports us.

The idea behind Stirling-pdf is no tracking and all free so it's hard to say sadly

**Comment by Frooodle** [comment]
I would be very interested if companies could come forward with this info

**Comment by PreviST** [comment]
We're looking into the possibility of offering it to our users to replace in-house tools based on PDFtk and to replace Adobe Acrobat / PDFsam for very simple uses. We're currently testing it.
What we're most often asked for is the ability to visually delete several pages in a PDF, which we can't do with PDFtk, but with this tool it's possible. Our users manage by using external sites such as ILovePDF, which we block when we become aware of them. 
We're a French company with around 500 employees.

**Reply by Frooodle** [reply]
Let me know if your team faces any issues and we can try resolve them!

**Comment by HydrelioxGitHub** [comment]
I try also to test Stirling in my company. It's a public urban community (I don't know if it's the right term in English, sorry). We have around 300 employees. I've made a VM available in our local network only for IT service for now. The main purpose is to convert from and to doc and docx file format. I have not communicate a lot about it as it's a test.

I plan to migrate to cloudron or runtipi as several tools are interesting for us (not only Stirling) and our system admin do not have time to handle those.

**Comment by saitoh183** [comment]
@Frooodle Im also going to talk to my manager about giving this a try in our company, Around 1000 employees/13 offices across Canada.  I was wondering if it would be possible to make this into a sort of default viewer of sorts (like edge) so that users dont have to open the site, then load up the pdf into the chosen tool so they can work on it. If you could associate it to pdf so it opens directly in the webpage when you double click on the pdf, it might make it easier for people to integrate it into there workflow. My only worry is that users that modify PDFs on a daily basis find it painful to load the webpage, then click on the tool they need, then load the pdf into it. That is alot of steps when you have like 50 pdfs to modify. We currently use Adobe Acrobat and PDF Xchange and this tool seems like the perfect solution to replace both.

**Reply by Frooodle** [reply]
So currently there is no support for this, and sadly there will not be for a while.

However, some other features that might interest you that we are working on which you might find useful and thats running multiple pre-defined operations on a PDF at once which should lessen some of the pain they have to go through.
Example here
![image](https://github.com/Stirling-Tools/Stirling-PDF/assets/77850077/d3b4b206-96f8-4083-8a4c-129a97840d49)

**Reply by Frooodle** [reply]
Also if the app does prove useful for you I would greater appreciate it if you could consider just donating just 10% of the money we would save you.
Also we plan to offer business support models in the future should it interest you

**Reply by saitoh183** [reply]
Thanks for the info. And yes I will definitely mention the donation if we decide to implement after testing.

**Reply by Frooodle** [reply]
Additionally if after testing you deem it's not for you could you please give feedback on the missing features you require so hopefully we can enhance in future

**Reply by tymanthius** [reply]
@saitoh183 just a thought on maybe helping your users - create bookmarks on their desktops with Stirling icon's to the most commonly used tools.  That would reduce the number of clicks at least.

**Comment by Peter-TLF** [comment]
I'm working on offering it at the Law Firm I work at, non-profit civil litigation law firm, often have a large volume of law school interns, so not feasible to try and have adobe or pdf same access for all of them, and most people in the legal field don't have a high grasp of technology, but the paralegals I have shown this to are very impressed. I know one thing that is highly sought after in any pdf tool I recommend to them is Bates Numbering, with batch redacting of HIPPA Forms, being a close second.

**Comment by ibyteprettyhard** [comment]
I just came across this project and already have a lot of ideas to implement it in my company.  The only thing I foresee as a barrier is no oauth functionality.  In order for me to offer this outside of a small group of testing folk, I'd need the ability to have it as an option within the Office365 "waffle".  This would allow the user who is already authenticated to access the service without having to login again; I have no interest in maintaining 2000+ accounts manually.  Perhaps an option to remove the login function completely so the user would then just need the proper internal URL, as long as there are no artifacts that persist between users.

**Reply by sbplat** [reply]
There is an option to disable login. It should be `security.enableLogin` in `/configs/settings.yml`.

**Reply by Frooodle** [reply]
It is also disabled by default

**Reply by igbjnI05bF** [reply]
I would love openid support. My instance is accessible over the Internet so I'd rather not disable auth entirely. I'd just very much like for it to integrate with my Keycloak so everybody in my small company doesn't need yet another account.

**Comment by Rolzzz** [comment]
hello, I'm looking to get it into our organisation. Internationally we're around 1600 users. For now I want to put one instance into my APAC region thats has about 300 users. UK is about 1000, how does this scale? Anyone have any experience there?

**Reply by Frooodle** [reply]
For Login/SSO platforms We scale with our enterprise license, The enterprise license is needed to externalise the database to allow distributed loading across multiple instances.
If without login you should be able to use it without

both implementations would require a load balancer between multiple installations

**Comment by Frooodle** [comment]
@dabump @saitoh183 @tymanthius @HydrelioxGitHub @PreviST @igbjnI05bF @Rolzzz 
We have recently introduced 
https://github.com/Stirling-Tools/Stirling-PDF/blob/main/USERS.md
In hopes to learn our community better! Please consider adding your company!
